README.md

```diff
@@ -315,3 +316,63 @@ Comp 40 gives you the tools to explore a large number of other topics:
+* Become acquainted with data structures. Learn how they work and draw them out with the TAs. Seek the TAs to learn how the complex data structures work instead of seeking them solely for debugging! The TAs for comp15 are an awesome group and all have a very deep understanding of how the data structures work. You will see that even one wrong detail in data structures will lead to segmentation fault quicksand or unexpected behavior. Each code you write really matter, so even if you understand, go check with the TAs to make sure you are crystal clear!
+* Decisions, decisions, decisions! There are multiple right answers in data structures, which is why, “am I doing it right?” will give you a very mixed answer in this class. Taking ownership of your decision after evaluating different options is a hallmark of a good engineer, so make sure you have a deep understanding of each data structure you cover in class. There is nothing minor that you can skip in this class, as each new data structure is now a new option you can consider for your upcoming homework or project. Each data structure has its own pros and cons, so learn how to evaluate and choose what you believe is the best for the problem you are trying to solve! 
+* Make sure you go to class. Mark does not use a powerpoint, so if you miss one, then you will not have any resources to catch up with class work. So if something happens, make sure you get note from someone and try to never fall behind because the class goes by quick!
+* There is no more HalliganHelper, but Piazza is always on to communicate with the TAs. Identify the location of the TAs through Piazza to get help! Also, Mark holds regular office hours and he is of great help if you want a deep understanding of the concepts discussed in class. From my personal experience, he normally takes around 20~30 minutes to explain concepts each time I visited, so if you are in line, you may have to wait quite a time. But once it is your turn, you will get the thorough help that you may need. Mark is awesome during office hours, so do try to take advantage of it!
+* Comp 15 debugging tutorial 101:
+  * First, you should run your code through a debugger (gdb) and/or valgrind. If you don't know how to use one or both of these tools, it is absolutely worth your time to meet with a TA at the beginning of the semester to get an introduction to these tools. Memory management is new to all comp 11 students and it is essential you learn how to use these tools. You won’t survive without it, so learn it ASAP!
```

Comment:
Hello Graham! 

I modeled the comp15 guideline after reading yours on comp40 and thought that debugging technique were similar to both. And because I didn't think people reading comp15 material would read comp40 material, I took much of the details you have previously written. However, I can try to tailor it towards the challenges students face in comp15.

I also liked your comment about the student's question and want to link it if possible. However, when I searched into it, there seems to be no option to link to lines.
https://github.com/holman/ama/issues/305

Could we make a common section for that? Maybe a guideline on how to interact with the TA and debug?